geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisHash.java

```diff
@@ -58,6 +59,9 @@
   private ConcurrentHashMap<UUID, List<ByteArrayWrapper>> hScanSnapShots;
   private ConcurrentHashMap<UUID, Long> hScanSnapShotCreationTimes;
   private ScheduledExecutorService HSCANSnapshotExpirationExecutor = null;
+  private static final int PER_STRING_OVERHEAD = PER_OBJECT_OVERHEAD + 46;
+  private static final int PER_HASH_OVERHEAD = PER_OBJECT_OVERHEAD + 116;
+  private AtomicInteger hashSize = new AtomicInteger(PER_HASH_OVERHEAD);
```

Comment:
Does this need to be an `AtomicInteger` or will just an `int` do? I see all of our update methods are synchronized, and I don't think the HashMap is safe under concurrent modification.